src/Helper/AbstractHtmlElement.php

```diff
@@ -64,7 +64,7 @@ protected function htmlAttribs($attribs)
             }
         }
 
-        $attribs = $this->getView()->plugin(HtmlAttributes::class)($attribs);
+        $attribs = $this->getView()->plugin('htmlattributes')($attribs);
```

Comment:
IMO this is not valid: if you wanted to have a custom plugin used, then the right approach would be to move thus to `__construct`, and then override how this specific helper is built.